frame/transaction-payment/src/lib.rs

```diff
@@ -679,6 +679,22 @@ where
 		}
 		.saturated_into::<TransactionPriority>()
 	}
+
+	pub fn do_post_dispatch(
```

Comment:
You will need to document this, though, if you introduce another `pub` function

Reply:
Not needed anymore.